docker/nodes.go

```diff
@@ -143,17 +143,22 @@ func (n *Node) Create(cloudConfig []byte) (*nodes.Node, error) {
 		// Node length includes ELB which must already exist
 		if len(n.Nodes) == 1 {
 			log.Info("Creating the first control plane node")
-			node, err := nodes.ExternalLoadBalancerNode(n.Nodes)
+			cp, err := n.MachineActions.CreateControlPlane(n.Cluster, n.Machine, n.Version, nil, cloudConfig)
 			if err != nil {
-				log.Error(err, "Failed to get external load balancer node")
-				return nil, err
+				return nil, errors.Wrap(err, "failed to create control plane")
 			}
-			ipv4, _, err := node.IP()
+
+			// save the kubeconfig on the host with the loadbalancer endpoint
+			dest := actions.KubeConfigPath(n.Cluster)
+			hostAddress := "127.0.0.1"
```

Comment:
Is this right? I think remoteutil will need the container address, not localhost

Reply:
AFAIK as I know this writes the kubeconfig for the remote cluster on the host machine.

Writing the kubeconfig file in the management cluster is still an open point; see https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm/issues/143 and https://github.com/kubernetes-sigs/cluster-api/issues/1274

Reply:
ah got it ok

Reply:
Do you think we even need to do this? It seems like a nice-to-have versus something required for the system.